.travis.yml

```diff
@@ -22,4 +22,4 @@ matrix:
             - powershell
 
 script:
-  - pwsh -c 'Import-Module ./Pester.psd1; Invoke-Pester -ExcludeTag VersionChecks -EnableExit'
+  - pwsh -c 'Import-Module ./Pester.psd1; Invoke-Pester -ExcludeTag VersionChecks, StyleRules -EnableExit'
```

Comment:
This will have effect only for builds on Linux and macOS.

Reply:
I know.